src/java.base/share/classes/jdk/internal/classfile/impl/StackMapGenerator.java

```diff
@@ -180,7 +181,21 @@ static StackMapGenerator of(DirectCodeBuilder dcb, BufWriterImpl buf) {
             ITEM_SHORT = 11,
             ITEM_CHAR = 12,
             ITEM_LONG_2ND = 13,
-            ITEM_DOUBLE_2ND = 14;
+            ITEM_DOUBLE_2ND = 14,
+            ITEM_BOGUS = -1;
+
+    public static final int
+            SAME_FRAME_END = 63,
```

Comment:
Can you add a comment on these constants that they are inclusive on both ends? A lot of ranges in Java are open on their higher end, like range notations in Arrays.copyOf or String.substring.

Reply:
Sure: 0c6c04b7b8874d3b3bd53c4f9456fe01b151241d